x-pack/test/functional_with_es_ssl/apps/triggers_actions_ui/alerts.ts

```diff
@@ -0,0 +1,452 @@
+        consumer: 'alerts',
+        schedule: { interval: '1m' },
+        throttle: '1m',
+        actions: [],
+        params: {},
+        ...overwrites,
+      })
+      .expect(200);
+    return createdAlert;
+  }
+
+  async function getAlertsByName(name: string) {
+    const {
+      body: { data: alerts },
+    } = await supertest.get(`/api/alerts/_find?search=${name}&search_fields=name`).expect(200);
+
+    return alerts;
+  }
+
+  async function deleteAlerts(alertIds: string[]) {
+    alertIds.forEach(async (alertId: string) => {
+      await supertest.delete(`/api/alerts/alert/${alertId}`).set('kbn-xsrf', 'foo').expect(204, '');
+    });
+  }
+
+  async function defineAlert(alertName: string) {
+    await pageObjects.triggersActionsUI.clickCreateAlertButton();
+    await testSubjects.setValue('alertNameInput', alertName);
+    await testSubjects.click('.index-threshold-SelectOption');
+    await testSubjects.click('selectIndexExpression');
+    const comboBox = await find.byCssSelector('#indexSelectSearchBox');
+    await comboBox.click();
+    await comboBox.type('k');
+    const filterSelectItem = await find.byCssSelector(`.euiFilterSelectItem`);
+    await filterSelectItem.click();
+    await testSubjects.click('thresholdAlertTimeFieldSelect');
+    await retry.try(async () => {
+      const fieldOptions = await find.allByCssSelector('#thresholdTimeField option');
+      expect(fieldOptions[1]).not.to.be(undefined);
+      await fieldOptions[1].click();
+    });
+    await testSubjects.click('closePopover');
+    // need this two out of popup clicks to close them
+    const nameInput = await testSubjects.find('alertNameInput');
+    await nameInput.click();
+  }
+
+  describe('alerts', function () {
+    before(async () => {
+      await pageObjects.common.navigateToApp('triggersActions');
+      await testSubjects.click('alertsTab');
+    });
+
+    it('should create an alert', async () => {
+      const alertName = generateUniqueKey();
+      await defineAlert(alertName);
+
+      await testSubjects.click('.slack-ActionTypeSelectOption');
+      await testSubjects.click('addNewActionConnectorButton-.slack');
+      const slackConnectorName = generateUniqueKey();
+      await testSubjects.setValue('nameInput', slackConnectorName);
+      await testSubjects.setValue('slackWebhookUrlInput', 'https://test');
+      await find.clickByCssSelector('[data-test-subj="saveActionButtonModal"]:not(disabled)');
+      const createdConnectorToastTitle = await pageObjects.common.closeToast();
+      expect(createdConnectorToastTitle).to.eql(`Created '${slackConnectorName}'`);
+      const messageTextArea = await find.byCssSelector('[data-test-subj="messageTextArea"]');
+      expect(await messageTextArea.getAttribute('value')).to.eql('alert {{alertName}} group {{context.group}} value {{context.value}} exceeded threshold {{context.function}} over {{params.timeWindowSize}}{{params.timeWindowUnit}} on {{context.date}}');
```

Comment:
This is the test I was expecting to see.  However, we don't need a entire new test module for this, we can just put these two lines into an existing test; I think right before this line: https://github.com/elastic/kibana/blob/fb629bd94d4b721e0151736a33f5a2492d4d94d1/x-pack/test/functional_with_es_ssl/apps/triggers_actions_ui/alert_create_flyout.ts#L76

I didn't significantly look through this test though.  Was there some other reason we needed a new test module, vs adding the check to an existing test module?

Reply:
@pmuellr Should I put those 2 lines here?
https://github.com/elastic/kibana/blob/fb629bd94d4b721e0151736a33f5a2492d4d94d1/x-pack/test/functional_with_es_ssl/apps/triggers_actions_ui/alert_create_flyout.ts#L76

Reply:
yes, seems like you can put the test for the default message just before that line, where it will then get reset by the test.

Reply:
@pmuellr Done.

Reply:
yes, sorry for the delay, I don't _think_ that file is needed, unless you added more tests beyond the one I suggested for `alert_create_flyout.ts`

Reply:
@pmuellr I have committed the changes you suggested.